web-local/src/routes/(viz)/explore/[name]/+page.svelte

```diff
@@ -16,20 +15,21 @@
   const queryClient = useQueryClient();
 
   export let data: PageData;
+  $: ({ metricsView, explore, basePreset, partialMetrics } = data);
```

Comment:
It's confusing seeing both `basePreset` and `partialMetrics`. Could we merge the two of them further upstream?

Reply:
They are not used together. If `basePreset` is used to keep the url clean and not add defaults. `partialMetrics` is the metrics from url. `basePreset` will not be needed in the future when we directly update the url in actions.